x-pack/test_serverless/functional/config.base.ts

```diff
@@ -27,6 +27,7 @@ export function createTestConfig(options: CreateTestConfigOptions) {
         serverArgs: [
           ...svlSharedConfig.get('kbnTestServer.serverArgs'),
           `--serverless=${options.serverlessProject}`,
+          '--xpack.encryptedSavedObjects.encryptionKey="wuGNaIhoMpk5sO4UBxgr3NyW1sFcLgIf"',
```

Comment:
Might be worth to move this to https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/shared/config.base.ts so it's effective for functional testa dna API integration tests.

Reply:
API integration config already has it. So I will remove it from API and functional config and put it in a shared config.